handout/forms/control_grouping.md

```diff
@@ -2,7 +2,7 @@
 
 In Angular 2, there are three basic built-in classes that help us to manage forms:
 
-- `Control` - A [Control](https://angular.io/docs/ts/latest/api/common/Control-class.html) is the most basic unit of a form, associated with individual fields of a form. Through a `Control` we can know the field's value, its state (whether or not is valid or has been changed), and its errors.
+- `Control` - A [Control](https://angular.io/docs/ts/latest/api/common/Control-class.html) is the most basic unit of a form, associated with individual fields of a form. Through a `Control` we can know the field's value, its state (whether or not is valid or has been changed) and its errors.
```

Comment:
Something weird in the parenthetical. Perhaps it should instead be: (whether it's valid or has been changed)